RELEASE.md

```diff
@@ -0,0 +1,200 @@
+
+    ```
+    $ git clone git@github.com:habitat-sh/habitat.git ~/code/habitat
+    ```
+
+1. Clone the core-plans repository into a subdirectory of your code directory
+
+    ```
+    $ git clone git@github.com:habitat-sh/core-plans.git ~/code/core-plans
+    ```
+
+1. [Install Habitat for Mac](https://www.habitat.sh/docs/get-habitat/)
+1. Install AWS CLI
+
+    ```
+    $ brew install awscli
+    ```
+
+1. Setup an AWS profile in `~/.aws/credentials` named `habitat` with your `aws_access_key_id` and `aws_secret_access_key`
+
+    ```
+    $ cat ~/.aws/credentials
+    # ~~~~~~~~~~~~~~~~~~~
+    # Your other profiles
+    # ~~~~~~~~~~~~~~~~~~~
+
+    [habitat]
+    aws_access_key_id=$(KEY_ID)
+    aws_secret_access_key=$(SECRET_KEY)
+    region=us-west-2
+    ```
+
+1. Download OSX 10.9 Vagrant Box
+
+    ```
+    $ aws s3 cp --profile habitat s3://habitat-initial-hab/macosx-10.9-2.1.20160526021718.git.6fdd2e497a8fc2999c4195bab9f3f5127dd70d6a.vmware.box .
+    ```
+
+1. Install [Vagrant](https://www.vagrantup.com/downloads.html)
+1. Install [VMWare Fusion](http://www.vmware.com/products/fusion/fusion-evaluation.html)
+1. [Purchase VMWare Fusion License](https://www.vagrantup.com/vmware/#buy-now)
+1. Install [Vagrant VMWare Fusion Provider & License](https://www.vagrantup.com/docs/vmware/installation.html)
+1. Install OSX 10.9 Vagrant Box
+
+    ```
+    $ vagrant box add bento/macosx-10.9 macosx-10.9-2.1.20160526021718.git.6fdd2e497a8fc2999c4195bab9f3f5127dd70d6a.vmware.box
+    ```
+
+## Prepare Master Branch for Release
+
+1. Ensure you are on the master branch and have the latest of `~/code/habitat` and `~/code/core-plans`
+
+    ```
+    $ cd ~/code/habitat
+    $ git checkout master
+    $ git pull origin master
+    $ cd ~/code/core-plans
+    $ git checkout master
+    $ git pull origin master
+    ```
+
+1. Create a new release branch in the Habitat repo
+
+    ```
+    $ cd ~/code/habitat
+    $ git checkout -b <branch>
+    ```
+
+1. Update the contents of `VERSION` with the release's new version number
+
+    ```
+    $ vi ~/code/habitat/VERSION
+    ```
+
+1. Generate a new `CHANGELOG.md`
+
+    ```
+    $ export GITHUB_TOKEN=<your-token>
+    $ make changelog
+    ```
+
+1. Commit the `CHANGELOG.md` and `VERSION` changes and push your branch    
+1. Issue a new PR await approval (in the form of a [dank gif](http://imgur.com/X0sNq)) from two maintainers
+1. Pull master once again once the PR is merged into master
+1. Create & push a Git tag
+
+    ```
+    $ make tag-release
+    $ git push origin --tags
+    ```
+
+## Build Release
+
+1. Fetch the tags from the upstream Habitat repository and checkout the release tag
+
+    ```
+    $ cd ~/code/habitat
+    $ git fetch origin --tags
+    $ git checkout <version>
+    ```
+
+1. Change to your code directory and enter a studio
+
+    ```
+    $ cd ~/code
+    $ hab studio enter
+    ```
+
+1. [Build each component](#how-to-build-a-linux-component)
```

Comment:
From within a studio you use `build` - in the build each component section below I go over the steps for building a component. Should I clarify somehow here that the person should go-to the section for instructions instead of just making this a link?